alpheios_nemo_ui/data/templates/main/container.html

```diff
@@ -56,45 +57,67 @@
     {% include "main::footer.html" %}
   {% endblock %}
   <script type="text/javascript">
-    document.addEventListener("DOMContentLoaded", function(event) {
-      import ('/assets/nemo.secondary/js/alpheios-embedded.min.js').then(embedLib => {
-        console.info(`Embedded library has been imported successfully`)
+    window.onerror = function(message) {
+      var parser = new UAParser();
+      var browserData = parser.getBrowser();
+      var allowedBrowsers = ['Chrome', 'Firefox', 'Mobile Safari', 'Safari'];
 
-        window.AlpheiosEmbed.importDependencies(
-          { mode: 'custom',
-            libs: { components: '/assets/nemo.secondary/js/alpheios-components.min.js' } }).then(Embedded => {
-          const embedded = new Embedded(
-            {
-               clientId:'alpheios-reader',
-               enabledSelector:'.entry-content',
-               popupInitialPos: {
-                 left: '50px',
-                 top: '230px'
-               },
-              toolbarInitialPos: {
-                top: '220px',
-                right: '15px'
-              },
-              layoutType: 'readingTools',
-              {% if collections['current'] and collections['current']['id'] and 'greek' in collections['current']['id'] %}
-              textLangCode: 'grc'
-              {% else %}
-              textLangCode: 'lat'
-              {% endif %}
-            })
-          embedded.activate().then(function () {
-            import ('/assets/nemo.secondary/js/alpheios-embed-support.js')
-              .then(m => m.embedPostActivation(embedded))
-              .catch(e => console.error(`There is an error importing alpheios-embed-support.js:`, e))
+      if (allowedBrowsers.indexOf(browserData.name) === -1) {
+        console.error('Your browser is not supported by Alpheios Extension. Please use Chrome, Firefox or Safari.')
+        window.alert('Your browser is not supported by Alpheios Extension. Please use Chrome, Firefox or Safari.')
```

Comment:
Minor change, let's word this as "Alpheios may not work with your browser.  Please use Chrome, Firefox or Safari to access this site."

Reply:
Fixed